tasks/files-to-granules/index.js

```diff
@@ -0,0 +1,112 @@
+'use strict';
+
+const get = require('lodash.get');
+const path = require('path');
+
+const { getGranuleId } = require('@cumulus/cmrjs');
```

Comment:
Since you're only using a couple of small functions from `@cumulus/cmrjs` and `@cumulus/commmon`, you might want to consider just re-implementing those functions here instead of importing those two packages.  I did a quick test and that takes the webpacked `dist/index.js` from 5 MB down to 98K.

That points at a bigger issue that we should probably address at some point (completely unrelated to this PR).  We have a lot of unrelated stuff piled into our packages (expecially `@cumulus/common`) so that, if I want a utility function like `isNil()` from the common package, I have to install the entire `aws-sdk` package.